lib/datasource/git-refs/index.ts

```diff
@@ -0,0 +1,50 @@
+import simpleGit from 'simple-git/promise';
+import { logger } from '../../logger';
+import { ReleaseResult, GetReleasesConfig } from '../common';
+
+export const id = 'git-refs';
+
+const cacheNamespace = 'git-refs';
+const cacheMinutes = 10;
+
+// git will prompt for known hosts or passwords, unless we activate BatchMode
+process.env.GIT_SSH_COMMAND = 'ssh -o BatchMode=yes';
```

Comment:
This should be moved to a more general file.

@rarkins @JamieMagee  Any opinion?

Reply:
It will have global effect, so maybe we should do it during global init?

Reply:
You mean into constants?

Reply:
no, i think here:
https://github.com/renovatebot/renovate/blob/1d661b909b18b5a09dc4b099c72cbd5397a6bb57/lib/workers/global/index.ts#L63-L69

And we shouldn't override if already set.

Reply:
okay but in another PR right? @viceice

Reply:
A bit late to the party, but I think moving this from `git-refs` and `git-tags` is the best option. You can do it now, or in another PR. I have no preference.

This did send me down a bit of a rabbit hole about what args we should set for git :smile:

Reply:
@JamieMagee do you mean moving the `process.env` part? And why in `git-tags`?

Reply:
@rarkins Yes, moving `process.env`. And this same line of code also exists in `git-tags`. We should set it once during initialisation.

Reply:
we can keep it in `git-refs` and remove it from `git-tags` as `git-tags` is now using `git-refs` code and also `git-refs` executes first @JamieMagee